test/integration/functional_test.go

```diff
@@ -1539,16 +1540,10 @@ func validateServiceCmdFormat(ctx context.Context, t *testing.T, profile string)
 			t.Errorf("failed to get service url with custom format. args %q: %v", rr.Command(), err)
 		}
 
-		endpoint := strings.TrimSpace(rr.Stdout.String())
-		t.Logf("found endpoint for hello-node: %s", endpoint)
-
-		u, err := url.Parse(endpoint)
-		if err != nil {
-			t.Fatalf("failed to parse %q: %v", endpoint, err)
-		}
+		stringIP := strings.TrimSpace(rr.Stdout.String())
 
-		if strings.TrimSpace(rr.Stdout.String()) != u.Hostname() {
```

Comment:
would the hostname always be ip ? or would it be sometimes like svc-name.default.local.svc ?

Reply:
It's always an IP, it's either `minikube ip` or the IP of the SSH tunnel (which is `127.0.0.1`)